tests/BehatFeatures/web/translation/credits_custom_translation.feature

```diff
@@ -8,96 +8,97 @@ Feature: Projects should have credits where a custom translation can be defined
     And there are projects:
       | id | name      | owned by | credit     |
       | 1  | project 1 | Catrobat | my credits |
+      | 1  | project 2 | Catrobat |            |
     And I wait 1000 milliseconds
 
-  Scenario: Custom translation editor should be visible
-    Given I log in as "Catrobat"
-    And I go to "/app/project/1"
-    And I wait for the page to be loaded
-    When I click "#edit-credits-button"
-    And I wait for AJAX to finish
-    Then the element "#credits" should not be visible
-    But the element "#edit-text" should be visible
-    And the element "#edit-text-ui" should be visible
-    And the element "#edit-language-selector" should be visible
-    And the element "#edit-submit-button" should be visible
-
   Scenario: Adding a custom credits translation
```

Comment:
this and the one in name_custom_translation can be delelted to keep consistent with description_custom_translation

Reply:
Deleted